amida-tech/hl7-telescoper-api-server
# Hl7 10 am signup endpoint

### Related JIRA tickets:
>Please enter the whole URL so we can just click/copy it
-https://jira.amida.com/browse/HL7-10

### What exactly does this PR do?
- Created a signup endpoint for users to create account
- Added checks for for duplicate emails, and usernames
-  Added conditions for email and password


### Do ANY of these changes introduce a breaking change? (in-scope or otherwise)

- No

### Manual test cases?
- First enter username, email, password in Postman and send post request to api/users/signup
- After creating user, try sending again to see error message
- Also try to use incorrect emails or passwords that are to short (<6 char)

### Any additional context/background?

- The user.test will only work the first time for now as once you create a user it cannot be created again

### Screenshots (if appropriate):
